Cargo.toml

```diff
@@ -14,11 +14,16 @@ edition = "2018"
 [lib]
 name = "kuchikiki"
 
+[[bench]]
+name = "main_bench"
+harness = false
+
 [dependencies]
 cssparser = "0.31"
 html5ever = "0.27.0"
 selectors = "0.25"
 indexmap = "2.2.6"
 
 [dev-dependencies]
-tempfile = "3"
+criterion = { version = "0.4", features = ["html_reports"] }
```

Comment:
version 0.5 please, to avoid `cargo audit` warnings.

Reply:
Gah. My IDE betrayed me.